eks-anywhere-common/Addons/Partner/Sysdig/sysdig.yaml

```diff
@@ -0,0 +1,52 @@
+---
+apiVersion: helm.toolkit.fluxcd.io/v2beta1
+kind: HelmRelease
+metadata:
+  name: sysdig
+  namespace: sysdig-agent
+spec:
+  chart:
+    spec:
+      chart: sysdig-deploy
+      reconcileStrategy: ChartVersion
+      sourceRef:
+        kind: HelmRepository
+        name: sysdig
+        namespace: flux-system
+      version: 1.67.5
+  interval: 30s
+  targetNamespace: sysdig-agent
+  values:
+    global:
+      sysdig:
+        region: us2         # us1 | us2 | us3 | us4 | eu1 | au1
+      kspm:
+        deploy: false       # Disabled by default to optimize resources. Activating it without the right cluster sizing could lead to instability
+      clusterConfig:
+        name: myclusterName # Place here a meaningful cluster name to identify it
+    agent:
+      tolerations: []   # Nullifying tolerations disallows the agents from being executed in the Control Plane
+      # resourceProfile: custom
```

Comment:
remove these commeted lines for better readability

Reply:
Per discussion with Manual we'd like to retain for user visibility of configuration options